src/audio/drc/Testbench_drc_lin2dB/typdef.h

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+#pragma once
+typedef signed char int8_t;
+typedef unsigned char uint8_t;
+typedef short int16_t;
+typedef unsigned short uint16_t;
+typedef int int32_t;
+typedef unsigned int uint32_t;
+typedef float real32_t;
+typedef double real64_t;
```

Comment:
These are all in stdint.h - do we need float or double atm ?

Reply:
Yes, #include <stdint.h> will cater for  float and double. I will remove float and double atm